redash/query_runner/cass.py

```diff
@@ -7,8 +7,9 @@
 logger = logging.getLogger(__name__)
 
 try:
-    from cassandra.cluster import Cluster, Error
+    from cassandra.cluster import Cluster
     from cassandra.auth import PlainTextAuthProvider
+    from cassandra import DriverException
```

Comment:
Hi @arikfr 
I got confused.
I didn't notice that I worked with cassandra-driver==3.4.0 on my test server.